### DISCUSSION TITLE
Speedometer just two digits (bug?)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#20301] [Q&A]
Hello. 

Is this a bug with the new speedometer or did I miss some settings? 
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/f3fb7de7-162e-4418-9dbf-2f3f4429df90"  width="300" height="auto"/> </td>
  </tr>
</table>

speed lower than 100 km/h (two digits) works fine

Version 4.7.17

**Comment by yuriiurshuliak** [comment]
I can't reproduce this issue on my end. Please try updating the app to the Beta version from [this link](https://osmand.net/docs/versions/nightly_versions) and check if the issue persists there.

OsmAnd~ 4.8.0#2748m, released: 2024-07-16

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/cbb566fb-6ee2-4cf8-baed-9b4f9272cc5c"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by dmpr0** [comment]
Each size of Speedometer widget has own max. text size and min. text size, try to select bigger size of the widget.

**Reply by ThomasMeister123** [reply]
Cant test it now (can run more then 99 km/h in my flat)... :) 

But... dont look like there where THREE digits fit in the box. Not in running mode on map an not in setting-preview
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/4d53fcce-2c19-4013-955d-d277d7f43a5b"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/84c44cb8-4b2e-4177-8c0d-477987b85a50"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by DmitryAlexei** [reply]
@ThomasMeister123 you can try navigation simulation https://osmand.net/docs/user/navigation/setup/route-navigation#simulated-navigation

**Reply by ThomasMeister123** [reply]
Same in every size.
Same OSMAnd-Version on an other mobile works fine (3 digits). Dont work on Sony Xperia M
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/db57b244-e28d-4493-ab83-f1a28bdd1b79"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by vshcherb** [reply]
I think it's a bug if there are no text size manipulation on system settings, though we would need to reproduce it to understand what's the underlying cause

**Comment by DmitryAlexei** [comment]
OsmAnd~ 4.8.0#2787m, released: 2024-07-18 I didn't managed to reproduce this issue (see screenshot) even when font size is set to maximum and display size is also set to max.

- I configured the smallest available in Android Studio "Small Phone" with resolution 720x1280. Speedometer widget size is set to big.

Need to notice, that according to specs, Sony Xperia M is a phone from 2014 with resolution 480x854 

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/ffb8a556-9dbd-4b4f-9923-66671882108c"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by ThomasMeister123** [reply]
Yes, a 10 year old mobile, JUST for Navigation...

Is there a change to fix the bug by myself? Any config-files for this?

**Reply by DmitryAlexei** [reply]
You can try to increase or decrease the system font size. Also, disabling bold text may help.

**Reply by ThomasMeister123** [reply]
OK, I more thought of changes in the app. Dont want to change the whole mobile.
Then I have to switch off the new gadget :(
